src/calls-cost.js

```diff
@@ -38,7 +64,23 @@
  * 
  * @returns {CallsResponse}  - Processed information
 */
-
-function callsCost(calls) { }
+    
+function callsCost(calls) {
+  let prices = 0;
+  const details = calls.map((item) => {
+    const price = ProcessedCall(item);
+    prices += price;
+    return price ? {
+      ...item,
+      cost: +price.toFixed(2),
+    } : null;
```

Comment:
This price ternary operator condition is OK if we assume we can get calls with duration 0.
Ternary operator is fine. But in some cases reduces readability. 

```ts
const price = ProcessedCall(item);
if(!price) return null;

prices += price;

return {
 ...item,
  cost: +price.toFixed(2),
} ;
```

This tecnique is called `fail fast`. The goal is return or throw at start before we process and spend resources.